bark/python_wrapper/models/behavior.cpp

```diff
@@ -387,7 +389,50 @@ void python_behavior(py::module m) {
             return new BehaviorNotStarted(
                 PythonToParams(t[0].cast<py::tuple>()));
           }));
+  
+  py::class_<BehaviorSafety, BehaviorModel,
+             shared_ptr<BehaviorSafety>>(m, "BehaviorSafety")
+    .def(py::init<const bark::commons::ParamsPtr&>())
+    .def("SetBehaviorModel", &BehaviorSafety::SetBehaviorModel)
+    .def("__repr__",
+      [](const BehaviorSafety& b) {
+        return "bark.behavior.BehaviorSafety";
+      })
+    .def(py::pickle(
+      [](const BehaviorSafety& b) {
```

Comment:
Does not serialize underlying behavior model (e.g. IDMLaneTracking) -> Segfault in Benchmarking...

Reply:
But the models will be set in the constructor to the default values?

Reply:
Yes, but we need serialization of them. Otherwise we can only benchmark the defaults.